docs/fast_data/faq/bootstrapping-fast-data.md

```diff
@@ -0,0 +1,154 @@
+
+While designing your Fast Data solution, is crucial that <ins>the message producer will assign the same key to events involving the same record</ins>: in this way, they will be assigned always to the same partition and then processed always in the same order.  
+
+:::tip Useful Resources
+For more detailed benchmarks about topic partitions and data distribution among it, you can refer to this [Confluent blog post](https://www.confluent.io/blog/how-choose-number-topics-partitions-kafka-cluster/) .
+:::
+
+### What is the topic setup I should use to guarantee an high availability?
+
+A topic is distributed among the brokers of your cluster, with a parameter called _replication factor_: for critical messages, this value should be set to the number of brokers, meaning that the data of a topic will be stored inside all brokers.
+
+You can also specify how many replicas are needed to send acknowledgments on produced messages before committing, to guarantee the synchronization of the distributed data.
+
+This value can be set at the topic level with the parameter `min.insync.replicas` and is usually equal to `# of brokers - 1`. Critical flows may require a number of in-sync replicas equal to the number of brokers. 
+
+## Mongo Db
+
+### What should be the required resources to perform an Initial Load?
+
+The **Initial Load** is a manual operation made at runtime to initialize the projection's records with the most recent copy of the dataset from the system of record linked to it.
+
+This operation is carried out when we need the more recent copy of a table, in order to transfer the data present within the Fast Data system, with a **Change Data Capture** (**CDC**) and will be then processed by the [Real-Time Updater (RTU)](/fast_data/realtime_updater.md).
+
+:::danger Before executing the Initial Load
+Since an high number of ingestion messages will be handled by the RTU, remember **to disable strategy** during the execution of the Initial Load.
+
+This will avoid to perform additional MongoDb queries made by the strategies to generate projection changes records.
+:::
+
+:::tip Execution Time
+An initial load of `6.000.000 records` took around _10 minutes_, which is approximately a rate of `60k message/minute`: for this message rate, up until 70% of CPU from a single MongoDb core was used.  
+:::
+
+### What should be the required resources to perform a Single View Refresh?
+
+The **Single View Refresh** is an operation performed to update all the projection changes of a single view: this activity is needed during the first release of a single view and every time a new feature is released in the environment, since it will start the aggregation pipeline of the [**Single-View Creator**. (**SVC**)](/fast_data/single_view_creator.md)
```

Comment:
Technically, a SV refresh is needed only when a feature should be _retroactive_ with existing SVs. On the practical side, most of the time customer needs new fields also on older SVs.

I just don't know whether to reach this level of detail 🤔